### DISCUSSION TITLE
Cannot install GrandOrgue on MacBook Pro

### DISCUSSION THREAD

**Discussion by chartviewer** [GrandOrgue/grandorgue#1945] [Help] [answered]
Hi all, 

I am running GrandOrgue since quite a while on a windows notebook. Now I got a new MacBook Pro and tried to install the software. But I am getting the following error message (translated from German):
"“GrandOrgue” cannot be opened because Apple cannot search for malware in it."

The OS is macOS Sonoma 14.5.

Do you have an idea?

Thanks for your help.

Best regards,

Thorsten

**Comment by vpoguru** [comment, marked as answer]
Hello Thorsten, this is a message you will receive if the application is not notarized by Apple. Please read the following information on how to run the application anyway: https://support.apple.com/en-us/102445.

And in German: https://support.apple.com/de-de/guide/mac-help/mh40616/mac

**Reply by chartviewer** [reply]
Hello vpoguru,

Thanks a lot for your help!

**Comment by MStraeten** [comment]
after copying the app to Application folder run `xattr -c /Applications/GrandOrgue.app` from terminal

**Reply by chartviewer** [reply]
Even better - works . Thanks a lot!
